examples/fuse_examples/imaging/classification/mnist/run_mnist.py

```diff
@@ -245,7 +251,7 @@ def run_infer(paths: dict, infer_common_params: dict):
 
```

Comment:
in lines 247, 248 we are using a global ver `TRAIN_COMMON_PARAMS` which works but I think we better use the local `infer_common_params` (values should pass between dicts first)

Reply:
Already fixed it and missed a push, thanks